config/locales/en.yml

```diff
@@ -315,6 +315,9 @@ en:
       members:
         removed: "%{username} was successfully removed from team"
         added: "%{username} was successfully added to team"
+      favourites:
+        removed: "%{team_name} was successfully removed from the favourites"
+        added: "%{team_name} was successfully added to the favourites"
```

Comment:
Perhaps remove the. I think it increases language flow.

Reply:
We actually don't need these locales anymore, I agree with you though :p